assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -507,6 +518,13 @@ function Carousel() {
           </MenuIconsWrapper>
         </MenuArea>
       </Wrapper>
+      <EditorVersion>
+        {sprintf(
+          /* translators: %s: editor version. */
+          __('version %s', 'web-stories'),
```

Comment:
While at it: can you capitalize `Version` and make the same string translatable in the Dashboard too?

https://github.com/google/web-stories-wp/blob/aea3d8eb3fcef95f67c3e0cb993894eadda3de31/assets/src/dashboard/components/pageStructure/index.js#L230-L231

We should have the exact same `__('version %s', 'web-stories')` string in both locations, meaning it only needs to be translated once.

Reply:
Yep sorry, was copying what was in figma 
![figma-design](https://user-images.githubusercontent.com/2755722/91989658-c3db5100-ecfe-11ea-8d4c-f22a9de09171.png)

Reply:
Let's capitalize it still 🙂